rotkehlchen/chain/evm/decoding/weth/decoder.py

```diff
@@ -149,3 +149,30 @@ def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
     @staticmethod
     def counterparties() -> tuple[CounterpartyDetails, ...]:
         return (CounterpartyDetails(identifier=CPT_WETH, label='WETH', image='weth.svg'),)
+
+
+class WethDecoder(WethDecoderBase):
+    """
+        Weth Decoder for all supported EVM chains except Gnosis and Polygon Pos
+        because of different counterparty and not having ETH as native token.
```

Comment:
```suggestion
Weth Decoder for all supported EVM chains except Gnosis and Polygon Pos
because of different counterparty and not having ETH as native token.
```

no need to extra indent the comment